ngraph/test/files/paddlepaddle/models/models.csv

```diff
@@ -13,15 +26,48 @@ bilinear_upsample_scales,
 bilinear_upsample_scales2,
 bilinear_upsample_true_0,
 clip,
+conv2d_dilation_assymetric_pads_strides,
+conv2d_SAME_padding,
+conv2d_strides_assymetric_padding,
+conv2d_strides_no_padding,
+conv2d_strides_padding,
+conv2d_transpose_dilation_assymetric_pads_strides,
+conv2d_transpose_strides_assymetric_padding,
+conv2d_transpose_strides_no_padding,
+conv2d_transpose_strides_padding,
+conv2d_transpose_VALID_padding,
+conv2d_VALID_padding,
+depthwise_conv2d_convolution,
+depthwise_conv2d_transpose_convolution,
+dropout,
+expand_v2,
+expand_v2_tensor,
 fill_constant,
 fill_constant_batch_size_like,
 fill_constant_int32,
 flatten_contiguous_range_test1,
+maxAdaptivePool2D_test1,
+maxPool_test1,
+maxPool_test2,
+maxPool_test3,
+maxPool_test4,
+maxPool_test5,
+maxPool_test7,
+maxPool_test8,
+maxPool_test9,
+maxPool_test10,
+maxPool_test11,
 nearest_downsample_false_0,
 nearest_upsample_false_0,
 pad3d_test1,
 pad3d_test2,
 pad3d_test3,
+range,
+rnn_lstm_layer_1_bidirectional,
+rnn_lstm_layer_1_forward,
+rnn_lstm_layer_2_bidirectional,
+rnn_lstm_layer_2_forward,
+shape,
```

Comment:
Also not related to this PR, and very-very minor:

https://github.com/nosovmik/openvino/blob/d04a8667f56974214c8986c7de9e39bae52a0175/ngraph/test/files/paddlepaddle/models/models.csv#L32-L33

these lines are not in alphabetical order :-)